### PR TITLE
update launch web modeler page

### PR DESCRIPTION
**Pull request by christinaausley**
## Description

Closes https://github.com/camunda/developer-experience/issues/316 -- refreshes Launch Web Modeler documentation.

**Will backport upon approval.**

To do: Maybe add some next steps and link to the creating a process guide.

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [x] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [x] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

File: docs/components/modeler/web-modeler/launch-web-modeler.md [hunk 1]
```diff
@@ -1,29 +1,21 @@
...
 id: launch-web-modeler
 title: Launch Web Modeler
-description: "Since Web Modeler is embedded in the Camunda 8 Console, it does not need to be started separately. To launch Web Modeler, take these steps."
+description: "Design and implement your first diagram using Web Modeler, a component of Camunda where you can also create BPMN diagrams, DMN diagrams, or forms."
 ---
 
 <span class="badge badge--cloud">Camunda 8 only</span>
 
-Since Web Modeler is embedded in the Camunda 8 Console, it does not need to be started separately.
+Design and implement your first diagram using Web Modeler, a component of Camunda where you can also create BPMN diagrams, DMN diagrams, or forms.
```
Comment (akeller) [inline docs/components/modeler/web-modeler/launch-web-modeler.md:9]:
```suggestion
Design and implement your first diagram using Web Modeler, a component of Camunda, where you can also create BPMN diagrams, DMN diagrams, or forms.
```
My brain wants an additional comma here. WDYT? If so, please also add it to the metadata description.

File: docs/components/modeler/web-modeler/launch-web-modeler.md [hunk 2]
```diff
@@ -1,29 +1,21 @@
...
-   - In the details modal, you can find information about the author, the technical level, the description, and the categories of the selected template. You can also browse the diagram from the preview on the right side. To proceed with creating the diagram, click **Create model from template**.
-     ![web modeler template details modal](img/web-modeler-new-user-template-details-modal.png)
+1. After [creating an account and logging in](/guides/create-account.md) to Camunda, you are automatically taken to Web Modeler. Navigate between the components by clicking the square-shaped **Camunda components** icon in the top left corner of the page.
+2. Select **Create new project** to create a new project and store diagrams.
+   ![web modeler empty home](img/web-modeler-new-user-home.png)
+3. Name your diagram. You can go back and change the name any time by clicking on the project name and **Edit name**.
+4. Select **Browse blueprints** to view blueprints for various use cases as a starting point for your first diagram. Open these blueprints by selecting **Use Blueprint**. Alternatively, click **Create new > BPMN diagram** to create a blank BPMN diagrams.
```
Comment (akeller) [inline docs/components/modeler/web-modeler/launch-web-modeler.md:17]: Is it "blueprints" or "Blueprints" in the product? The casing should be consistent, but it's if inconsistent across the product let's be consistent **with** the product until that's changed.

File: docs/components/modeler/web-modeler/launch-web-modeler.md (hunk 1 above)
Comment (conceptualshark) [reply docs/components/modeler/web-modeler/launch-web-modeler.md:9]: I get this sense also - sometimes when I can't decide which version is easier to read, I'll just make two sentences for clarity?
```suggestion
Design and implement your first diagram using Web Modeler, a component of Camunda. Web Modeler also allows you to create BPMN diagrams, DMN diagrams, and forms.
```

File: docs/components/modeler/web-modeler/launch-web-modeler.md (hunk 2 above)
Comment (christinaausley) [reply docs/components/modeler/web-modeler/launch-web-modeler.md:17]: This is indeed how it is referenced in the product currently with the alternating casing -- happy to raise it with the Modeler team as far as UI goes, but matching it in the docs for now.

File: docs/components/modeler/web-modeler/launch-web-modeler.md (hunk 1 above)
Comment (christinaausley) [reply docs/components/modeler/web-modeler/launch-web-modeler.md]: Updated -- thank you both!

**Review by conceptualshark — APPROVED** [review]

File: docs/components/modeler/web-modeler/launch-web-modeler.md (hunk 2 above)
Comment (akeller) [reply docs/components/modeler/web-modeler/launch-web-modeler.md:17]: Thank you! Please raise it with them.

**Review by akeller — DISMISSED** [review]

**Comment by christinaausley** [comment]
This is backported to `8.5` and ready for a final review 👍

**Review by akeller — APPROVED** [review]

**Pull request merged**